styles/footer.css

```diff
@@ -0,0 +1,82 @@
+.footer-container {
+    justify-content: flex-start;
+    background-color: #6dc5dd;
+    box-sizing: border-box;
+    padding: 56px 0 1px 0;
+    border-radius: 0 0 5px 5px;
+}
+
+.footer {
+    justify-content: space-between;
+    color: white;
+    min-height: 260px;
+    height: 100%;
+}
+
+.footer__list {
+    width: 100%;
+    margin-bottom: 34px;
+
+    display: grid;
```

Comment:
Opt for one thing here. If you use grids, then use grids. If you use flexbox, then you need to place elements using flexbox. Both options are completely possible here. But a mix looks really weird.

Reply:
Well, I might have been referring to something different here. Nevermind, skip this.